hbase_sql/src/main/java/com/splicemachine/lifecycle/RegionServerLifecycle.java

```diff
@@ -84,13 +98,22 @@ public void distributedStart() throws IOException{
                     SQLConfiguration.upgradeForced = false;
 
                     // Ensure ZK paths exist.
-                    try{
+                    try {
                         ZkUtils.safeInitializeZooKeeper();
-                    }catch(InterruptedException e){
+                    } catch (InterruptedException e) {
                         throw new InterruptedIOException();
-                    }catch(KeeperException e){
+                    } catch (KeeperException e) {
                         throw new IOException(e);
                     }
+                } catch (TableExistsException e) {
+                    LOG.error("SPLICE_INIT already exists, this shouldn't happen. Remove it and retry");
```

Comment:
We may want to rewrite this to something like "SPLICE_INIT table exists. Removing it."